mod/metrics/set/right/checked_by.rb

```diff
@@ -76,7 +76,7 @@ def option_names
 end
 
 def answer
-  @answer ||= left&.answer
+  @answer ||= left.answer
```

Comment:
are you sure that we don't need the `&`?

Reply:
I'll put it back.  that was recently added  but didn't solve the problem (solved in `#unknown?`)